pyproject.toml

```diff
@@ -40,6 +40,10 @@ classifiers = [
     "Topic :: Internet :: File Transfer Protocol (FTP)",
 ]
 
+dependencies = [
+    "typing-extensions>=4.13.2",  # 3.9 typing support
```

Comment:
I think we need a TODO here: remove typing extension after 3.9 support drop. Am I right?

Reply:
Ok, add comment here then